assets/style/about/about.scss

```
@@ -6,9 +6,536 @@
+            .main__year_6 {
+              background: #2568e3;
+            }
+
+            .main__year-descr {
+              max-width: 215px;
+              font-weight: 400;
+              font-size: 16px;
+              line-height: 18px;
+              color: $text-blue-color;
+              @media (max-width: $tablet-s) {
+                font-size: 14px;
+              }
+              @media (max-width: $tablet-s) {
+                font-size: 13px;
+              }
+              @media (max-width: $phone-l) {
+                max-width: 90%;
+              }
+            }
+          }
+        }
+      }
+    }
+  }
+  &__wr-employees-partners {
+    background-color: #e5e5e5;
+    font-family: $font;
+    .main__employees-title,
+    .main__partners-title {
+      padding-top: 60px;
+      margin-bottom: 56px;
+      font-weight: 700;
+      font-size: 45px;
+      line-height: 46px;
+      text-align: center;
+      color: $text-blue-color;
+      @media (max-width: $tablet-l) {
+        font-size: 38px;
+        margin-bottom: 36px;
+      }
+      @media (max-width: $phone-m) {
+        font-size: 32px;
+      }
+    }
+    .main__wr-employees-img {
+      display: flex;
+      justify-content: space-between;
+      @media (max-width: $tablet-s) {
+        flex-wrap: wrap;
+      }
+
+      .main__employees-img {
+        display: flex;
+        flex-direction: column;
+        min-height: 556px;
+        justify-content: space-between;
+        max-width: 379px;
+        @media (max-width: $pc-s) {
+          min-height: 290px;
+        }
+        @media (max-width: $tablet-l) {
+          height: 290px;
+        }
+        @media (max-width: $tablet-s) {
+          flex-basis: 48%;
+          min-height: 380px;
+        }
+        @media (max-width: 410px) {
+          min-height: 300px;
+        }
+        img {
+          transform: scale(1.07);
+          margin-bottom: 15px;
+          @media (max-width: 1650px) {
+            display: block;
+            width: 100%;
+            height: auto;
+          }
+        }
+        .main__employees-name {
+          font-weight: 700;
+          font-size: 22px;
+          line-height: 32px;
+          color: $text-black-color;
+          @media (max-width: $tablet-s) {
+            font-size: 20px;
+          }
+          @media (max-width: $phone-s) {
+            font-size: 16px;
+          }
+        }
+        .main__employees-function {
+          font-weight: 400;
+          font-size: 20px;
+          line-height: 26px;
+          color: $text-blue-color;
+          margin-bottom: 112px;
+          @media (max-width: $pc-s) {
+            font-size: 18px;
+            max-width: 50px;
+            margin-bottom: 50px;
+          }
+          @media (max-width: $phone-s) {
+            font-size: 16px;
+          }
+        }
+      }
+    }
+
+    .main__partners-icons {
+      display: flex;
+      justify-content: space-between;
+      align-items: flex-end;
```

Зачем же flex-end? Я бы поставил center
```suggestion
      align-items: center;
```